app/views/organizations/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<% titleized_name = format_titleize org_name(@organization_officers) %>
+<% num_members = @organization_officers.length %>
+<% @layout_application_title = titleized_name %>
+<% @layout_application_description = "#{titleized_name} is an organization in the Boston Police Department with #{num_members} past or present members." %>
+
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <h1><%= titleized_name %></h1>
+      <p>This is an organization in the Boston Police Department. <%= link_to "Learn\u00a0more", page_path("help", anchor: "organizations") %></p>
+      <p>It has <%= num_members %> past or present <%= "member".pluralize(num_members) %>.</p>
+      <h3>Officers and employees in this organization</h3>
+      <div class="organization__table">
+          <%= render "shared/tables/officers", id: "officers-table", params: { organization: org_name(@organization_officers) }, options: {searching: false, saveState: false} %>
```

Comment:
That concern makes sense. Would a table of all the officers in a given organization rendered via the template be preferable? If so, I'm happy to make that update, since it would be a shame if this prevents us from getting the desired SEO benefit. 

No organization has more than 250 listed employees in the database, and most have way fewer, so there isn't much of an engineering need for paginated loading.

Reply:
Yeah, it would be optimal to render server-side:
1. the table of organizations on the organizations index page
2. the table of employees on the organization show page

but, I want to be respectful of your time -- this is acceptable as-is, and we can revisit later if rewriting this is going to take too much time right now